test/sagemaker_tests/pytorch/training/integration/sagemaker/test_pytorchddp.py

```diff
@@ -46,6 +46,7 @@ def can_run_pytorchddp(ecr_image):
 @pytest.mark.skip_py2_containers
 @pytest.mark.skip_trcomp_containers
 @pytest.mark.efa()
+@pytest.mark.skip_inductor_test
```

Comment:
Do we need to skip this test? It seems like there is a separate test for `test_pytorchddp_inductor.py`.